dolphinscheduler-ui/src/js/module/i18n/locale/zh_CN.js

```diff
@@ -420,7 +420,7 @@ export default {
   'Process instance details': '流程实例详情',
   'Create Resource': '创建资源',
   'User Center': '用户中心',
-  'Please enter method': '请输入方法',
+  AllStatus: '全部状态',
```

Comment:
Missing single quote?

Reply:
Thanks for your review, it seems the single quote is not required. Both the two kinds of writing exist in the project.

Reply:
How about keeping the same with others?

Reply:
Good suggestion